src/commonTest/kotlin/app/cash/turbine/FlowTurbineTest.kt

```diff
@@ -386,4 +386,37 @@ class FlowTurbineTest {
     channel.send("two")
     assertEquals(3, position)
   }
+
+  @Test fun expectLastItem() = suspendTest {
```

Comment:
Can we add a test where 1, 2 gets emitted and you expectMostRecentItem for 2, and then 3, 4 get emitted and you expect most recent for 4?

Probably can use a StateFlow or something with code like
```kotlin
test {
  stateFlow.emit(1)
  stateFlow.emit(2)
  assertEquals(2, expectMostRecentItem())

  stateFlow.emit(3)
  stateFlow.emit(4)
  assertEquals(4, expectMostRecentItem())

  cancelAndIgnore...()
}
```